bundles/test/org.eclipse.smarthome.test/src/main/java/org/eclipse/smarthome/test/OSGiTest.java

```diff
@@ -427,12 +428,15 @@ protected void setDefaultLocale(Locale locale) throws Exception {
         properties.put("variant", locale.getVariant());
 
         config.update(properties);
-        Closure closure = new Closure(null) {
+
+        waitForAssert(new Closure<Object>(null) {
+            private static final long serialVersionUID = -5083904877474902686L;
```

Comment:
Closure it marked serializable. Sure, not necessary here, but it doesn't matter...

Reply:
I was just about to fix those warnings and added a `@SuppressWarnings("serial")`. Works both.